server.go

```diff
@@ -82,6 +82,7 @@ type service struct {
 	server interface{} // the server for service methods
 	md     map[string]*MethodDesc
 	sd     map[string]*StreamDesc
+	meta   interface{}
```

Comment:
I don't think we'll agree here and since it's a unexported variable I don't care too much about it either. So even though I would prefer metadata, meta seems fine.